src/routes/safe/store/models/safe.ts

```diff
@@ -1,6 +1,25 @@
-import { List, Map, Record, Set } from 'immutable'
+import { List, Map, Record, RecordOf, Set } from 'immutable'
 
-const SafeRecord = Record({
+export type SafeRecord = {
```

Comment:
This is more like SafeRecordProps than SafeRecord :)

SafeRecord name is more suitable for this:
```
export type Safe = RecordOf<SafeRecord>
```

Reply:
oh, yes. You're right.